external/wakaama/Makefile

```diff
@@ -27,8 +27,9 @@ DEPPATH = --dep-path .
 VPATH =
 
 ifeq ($(CONFIG_LWM2M_WAKAAMA),y)
+CFLAGS += -I$(TOPDIR)/../external/wakaama/core
```

Comment:
artik053/wifi_test is working well without this.
Could you share what config do you use?

Reply:
there are couple of places which include path. so I removed it.